content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -39,6 +39,11 @@ Resource quotas work like this:
   See the [walkthrough](/docs/tasks/administer-cluster/manage-resources/quota-memory-cpu-namespace/)
   for an example of how to avoid this problem.
 
+{{< note >}}
+- For cpu/memory: ResourceQuota works only if every other pods in the namespace sets a limit for that resource. It means that you cannot create a new pod without limit/request CPU if the resource quota limits the CPU quota of this namespace.
+- For other resources: ResourceQuota works and will ignore pods in the namespace without setting a limit or request for that resource. It means that you can create a new pod without limit/request ephemeral storage if the resource quota limits the ephemeral storage of this namespace.
```

Comment:
This doesn't make sense to me. This sounds like the same behavior as for other resources.

If I define a Pod with no limit for memory and no request for memory, the Pod still works but can use up most of the memory of a node.
I f  defime a Pod with a container that mounts an `emptyDir` volume, and I don't limit the size of that volume, and I don't specify it as memory backed, I think the Pod creates OK even if there is a ResourceQuotsa for ephemeral storage in that Pod's namespace.

Does that sound right?

Reply:
>I f defime a Pod with a container that mounts an `emptyDir` volume, and I don't limit the size of that volume, and I don't specify it as memory backed, I think the Pod creates OK even if there is a ResourceQuotsa for ephemeral storage in that Pod's namespace.
> 
> Does that sound right?

Yes that's seems right the Pod will create and it will be in running state but the difference is ResourceQuota doesn't enforce in that case. We have to explicitly mentioned it in pod spec.

> If I define a Pod with no limit for memory and no request for memory, the Pod still works but can use up most of the memory of a node.

Pod can take up the memory limit and CPU limit up to which we have provided in resource Quota.

Reply:
What happens if I set a namespace-level limit for ephemeral storage, and then I submit a Pod that has no request and no limit for ephemeral storage, and that Pod defines an `emptyDir` volume?

Do cluster operators need to use LimitRange or a similar mechanism to make sure that resource quotas are enforced for resources other than `cpu` and `memory`?

Reply:
> What happens if I set a namespace-level limit for ephemeral storage, and then I submit a Pod that has no request and no limit for ephemeral storage, and that Pod defines an `emptyDir` volume?

This is allowed. Not allowed for cpu/memery. But allowed for ephemeral storage.

> Do cluster operators need to use LimitRange or a similar mechanism to make sure that resource quotas are enforced for resources other than `cpu` and `memory`?

In the case above, we have to suggest user to use LimitRange to add a default limit/request for ephemeral in Pod that has no request and no limit for ephemeral storage.

Reply:
OK. Let's make sure we recommend setting a LimitRange, then.

Reply:
> > What happens if I set a namespace-level limit for ephemeral storage, and then I submit a Pod that has no request and no limit for ephemeral storage, and that Pod defines an `emptyDir` volume?
> 
> This is allowed. Not allowed for cpu/memery. But allowed for ephemeral storage.

Yes this is allowed but the constraint is in that case ResourceQuota will not be enforced, and pod will be created successfully

Reply:
> > Do cluster operators need to use LimitRange or a similar mechanism to make sure that resource quotas are enforced for resources other than `cpu` and `memory`?
> 
> In the case above, we have to suggest user to use LimitRange to add a default limit/request for ephemeral in Pod that has no request and no limit for ephemeral storage.

This sounds ok to me to add LimitRange reference to it.